monitoring/space_monitor.py

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env python3
+
+"""
+Alert whenever more than 80% of the quota (or available space) is used.
+"""
+
+# ZFS concepts:
```

Comment:
Should the readme.rst be updated to this new script?

Reply:
Cool, I just add an example crontab entry.

Reply:
Good idea. Added!